portal-web/test/functional/com/liferay/portalweb/tests/enduser/documentmanagement/documentsadministration/DMLocalStaging.testcase

```diff
@@ -49,8 +49,6 @@ definition {
 	@description = "This is a test for LPS-91043. It asserts that user can not publish an old file entry version to live."
 	@priority = "5"
 	test CanNotPublishOldVerion {
-		property portal.acceptance = "true";
```

Comment:
@chaowang-123 we shouldn't add new tests to acceptance by default

Reply:
ok, got it, thanks!